app/services/employees/member_creator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Employees
+  class MemberCreator < ApplicationService
+    attr_reader :member_params, :company
+
+    def initialize(params, company)
+      super()
+      @member_params = params
+      @company = company
+    end
+
+    def call
```

Comment:
why do you add new logic during review? We don't need confirmation sending right now, and if we need it - we'll add a separate place for that. MemberCreator should only create member.